doc/source/io.rst

```diff
@@ -4475,6 +4475,15 @@ Additional information on service accounts can be found
 
 You will need to install an additional dependency: `oauth2client <https://github.com/google/oauth2client>`__.
 
+.. versionadded:: 0.19.0
+
+Authentication via ``application default credentials`` is also possible. This is only valid
```

Comment:
@jreback I don't completely understand this. Other authentication methods are already mentioned above this. I've just added the information for the new method. Could you please share the text that you want me to put here - if you have time. Thanks!